module/merintr/merintr.c

```diff
@@ -347,6 +348,8 @@ static TypedCommand commands[] = {
 { "tguild",      CommandTellGuild, },
 { "safetyon",    CommandSafetyOn, },
 { "safetyoff",   CommandSafetyOff, },
+{ "show time",   CommandTime, },
+{ "zeit anzeigen", CommandTime, }, // "show time" in German
```

Comment:
German translations don't go here---notice how there are no others in this list.